docs/04-resources/04-tutorials/index.md

```diff
@@ -13,67 +13,75 @@ values={[
 {label: 'Port to Rootstock', value: 'port-dapps'}
 ]}>
 <FilterItem
+value="beginner, advanced"
```

Comment:
Please move the guide to the quickstart section: https://dev.rootstock.io/developers/quickstart/

Vyper Starter Kit

Reply:
I have moved guide to quickstart section.